GetF!t/Modules/History/HistoryViewModel.swift

```diff
@@ -110,20 +109,21 @@ class HistoryViewModel: ObservableObject {
         }
     }
 
-    private func addDoneExercise(_ exerciseName: String) {
+    private func addDoneExercise(_ exerciseName: String, _ rating: Int) {
         let today = Date()
         if let firstDate = exerciseDays.first?.date,
            today.isSameDay(as: firstDate) {
             exerciseDays[0].exercises.append(exerciseName)
+            exerciseDays[0].rating.append(rating)
```

Comment:
tu by to potom vyzeralo takto:
```suggestion
            exerciseDays[0].exercises.append((exerciseName, rating))
```